var/spack/repos/builtin/packages/atmi/package.py

```diff
@@ -112,7 +114,10 @@ class Atmi(CMakePackage):
     root_cmakelists_dir = "src"
 
     patch("0001-Remove-relative-link-paths-to-external-libraries.patch", when="@3.5.0")
-    patch("0002-Remove-usr-bin-rsync-reference.patch", when="@4.0.0:")
+    patch("0002-Remove-usr-bin-rsync-reference.patch", when="@4.0.0:5.1.3")
+    # Reset the installation path and remove direct reference
+    # to /usr/bin/rysnc for rsync command.
+    patch("0002-Remove-usr-bin-rsync-reference-5.2.0.patch", when="@5.2.0:")
```

Comment:
Is the patch at line 117 any different than the one at line 120? If not you can:
```suggestion
    # Reset the installation path and remove direct reference
    # to /usr/bin/rysnc for rsync command.
    patch("0002-Remove-usr-bin-rsync-reference.patch", when="@4.0.0:5.1.3")
    patch("0002-Remove-usr-bin-rsync-reference-5.2.0.patch", when="@5.2.0:")
```
Looking at the `when=` it will be evident that they apply to different version ranges.

Reply:
In addition to the existing rsync direct reference fix in 0002-Remove-usr-bin-rsync-reference.patch
0002-Remove-usr-bin-rsync-reference**-5.2.0**.patch contains the fix for the atmi install path.
This has been fixed in 5.2 branch itself and the patch will no longer required in 5.2.1
rsync direct reference is tracking with atmi team to get addressed.